packages/manager/apps/pci-workflow/src/api/data/instance.ts

```diff
@@ -0,0 +1,13 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+
+export type TInstance = {
+  id: string;
+  name: string;
+};
+
+export const getInstance = async (projectId: string, instanceId: string) => {
```

Comment:
FYI: We have migrated the instances api calls into manager-pci-common. Once the common module is complete, we will proceed to use it.